files/usr/share/jeos-firstboot/modules/raspberrywifi

```diff
@@ -35,15 +35,23 @@ raspberrywifi_get_wlan_networks()
 {
 	list=()
 	local line
+	local has_hidden=
 	while read line; do
-		list+=("${line}" '')
-	done < <(ip link set $wlan_device up && iwlist $wlan_device scan|grep ESSID|cut -d':' -f2|cut -d'"' -f2|sort -u)
+	if [ -z $line ]; then
```

Comment:
IMO the `Enter SSID manually` entry could always be shown

Reply:
I guess it's better, because now testing it again, sometimes the iwlist misses the scan, so I will do that instead